stork-maven-plugin/src/main/java/com/fizzed/stork/maven/AssemblyMojo.java

```diff
@@ -9,11 +9,9 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugins.annotations.LifecyclePhase;
-import org.apache.maven.plugins.annotations.Mojo;
-import org.apache.maven.plugins.annotations.Parameter;
-import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.apache.maven.plugins.annotations.*;
```

Comment:
The project uses explicit imports -- can you please revert the code change which removed them and used a wildcard.

Reply:
My bad. Idea must have reformatted them. Will fix.